scripts/library.sh

```diff
@@ -418,6 +418,12 @@ function is_protected_gcr() {
   [[ -n $1 && "$1" =~ "^gcr.io/knative-(releases|nightly)/?$" ]]
 }
 
+# Return whether the given parameter is the knative-prow cluster.
+# Parameters: $1 - Kubernetes cluster context (output of kubectl config current-context)
+function is_protected_cluster() {
+  [[ -n $1 && "$1" = "gke_knative-tests_us-central1-f_prow" ]]
```

Comment:
I suggest that:
* `knative_tests` becomes a constant, used here and in `is_protected_project()`
*  you use a regex to ignore the zone (I think you can even ignore the region)